arangodb-net-standard/DatabaseApi/DatabaseApiClient.cs

```diff
@@ -48,5 +48,44 @@ public async Task<DeleteDatabaseResult> DeleteDatabaseAsync(string dbName)
                 throw new ApiErrorException(apiError);
             }
         }
+
+        /// <summary>
+        /// Retrieves the list of all existing databases.
+        /// (Only possible from within the _system database)
+        /// </summary>
+        /// <remarks>
+        /// You should use <see cref="ListUserDatabasesAsync"/> to fetch the list of the databases
+        /// available for the current user.
+        /// </remarks>
+        /// <returns></returns>
+        public async Task<ListDatabaseResult> ListDatabasesAsync()
```

Comment:
@rossmills99 Maybe this should be named `GetDatabasesAsync`?

Reply:
Yes, I think I agree

Reply:
Ok, I will create a new issue for that.